cypress-tests/cypress/e2e/happyPath/platform/eeTestcases/editmetadata.cy.js

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Consider including all CRUD operations in a single spec file instead of creating separate specs for each.